cloud/pkg/cloudhub/servers/httpserver/secretsutil.go

```diff
@@ -33,7 +33,7 @@ func GetSecret(secretName string, ns string) (*corev1.Secret, error) {
 func CreateSecret(secret *corev1.Secret, ns string) error {
 	cli := client.GetKubeClient()
 	if err := CreateNamespaceIfNeeded(cli, ns); err != nil {
-		return fmt.Errorf("failed to create Namespace kubeedge, error: %s", err)
+		return fmt.Errorf("failed to create Namespace kubeedge, error: %v", err)
```

Comment:
`%w` is better?

Reply:
from  https://github.com/golang/go/wiki/ErrorValueFAQ#i-am-already-using-fmterrorf-with-v-or-s-to-provide-context-for-an-error-when-should-i-switch-to-w

I think use %v is better for now

Reply:
LGTM